speedrunApiUtils.ts

```diff
@@ -46,7 +47,7 @@ export const getPlayerName = async (playerId: string) => {
 
 export const getCategory = async (categoryId: string) => {
   const categoryData = await fetch(
-    `https://www.speedrun.com/api/v1/categories/${categoryId}/records`
+    `https://www.speedrun.com/api/v1/categories/${categoryId}/records?${uuidv4()}`
```

Comment:
SRC API caches each request for around 10-15mins. To avoid this you can make each request unique (in this case by generating a random sequence of characters) to force the API to always give you up-to-date results.

Reply:
@Nick-NCSU how would you do it on this request? `https://www.speedrun.com/api/v1/categories/${categoryId}/records?status=verified`

is this legit? `https://www.speedrun.com/api/v1/categories/${categoryId}/records?status=verified&${uuidv4()}`
this just crashed with 
![image](https://user-images.githubusercontent.com/25116785/178851671-5ada054e-c3dc-45e8-92ff-083a0e2b9aa4.png)
![image](https://user-images.githubusercontent.com/25116785/178851671-5ada054e-c3dc-45e8-92ff-083a0e2b9aa4.png)

Reply:
That call seems to work fine for me...
https://www.speedrun.com/api/v1/categories/n2y55mko/records?status=verified&9999735cb-140c-4c94-8dfa-db0355d62913

Usually getting an `Unexpected token < in JSON at position 0` error means it returned an HTML response which can't be parse as JSON. SRC API does this usually when it returns a 504 error code. 
Example: https://www.speedrun.com/api/v1/games/k6q4rqzd?embed=categories.variables%2Clevels.variables
(this calls to embed every category, level, and subcategory in the game Seterra and usually always returns a 504).

Reply:
If you want to handle this error before parsing the JSON output I would do something like so:

```
export const getCategory = async (categoryId: string) => {
  const url = `https://www.speedrun.com/api/v1/categories/${categoryId}/records?status=verified&${uuidv4()}`
  const response = await fetch(url);

  if(!response.ok) {
    console.log(`Request returned ${response.status} from ${url}`);
    return;
  }
  
  const categoryData = await response.json();

  return categoryData;
};
```

or something along those lines. If the problem is happening repeatedly then let me know, but I think this may just be SRC's API having some issues.